### ISSUE TITLE
Should fail on RUN error

### ISSUE THREAD

**Issue by ctison** [earthly/earthly#204]
Earth version: `v0.3.0`

#### Earthfile
```
FROM alpine:3.12.0@sha256:90baa0922fe90624b05cb5766fa5da4e337921656c2f8e2b13bd3c052a0baac1
shouldfail:
  RUN false
  RUN echo KO
```

#### Result
```
buildkitd | Found buildkit daemon as docker container (earthly-buildkitd)
+base | --> FROM alpine:3.12.0@sha256:90baa0922fe90624b05cb5766fa5da4e337921656c2f8e2b13bd3c052a0baac1
+base | resolve docker.io/library/alpine:3.12.0@sha256:90baa0922fe90624b05cb5766fa5da4e337921656c2f8e2b13bd3c052a0baac1 100%
+shouldfail | --> RUN false
+shouldfail | Command /bin/sh -c  /bin/sh -c 'false' failed with exit code 1
+shouldfail | +shouldfail | --> RUN echo KO
+shouldfail | KO
+shouldfail | Target +shouldfail built successfully
=========================== SUCCESS ===========================
```

**Comment by alexcb** [comment]
Thank you very much for the report @ctison , we'll get this fix out ASAP.

**Comment by vladaionescu** [comment]
Hi @ctison - FYI, we just released `v0.3.1` of Earthly which contains a fix for this.
